OpenTrons/opentrons-api
# feat(app): add aria-disabled for displaying snack bar

# Overview
introduce `aria-disabled` to display a snack bar when the ODD displays `aria-disabled=true` button.

[design]
https://www.figma.com/design/MlZjOO8aZXsw3HfTBoyw9U/Release%3A-CSV?node-id=57-27179&t=ZOByYuCNeGTl4u5a-4

close AUTH-543


# Test Plan
1. send the attached protocol to a robot-server/Flex
2. setup a protocol
3. tap confirm values 
check the ODD shows the snack bar.


# Changelog



# Review requests



# Risk assessment
medium since this PR updates SmallButton
